build.sbt

```diff
@@ -6,12 +6,14 @@ ThisBuild / scalaVersion := "2.12.10"
 lazy val root = project
   .in(file("."))
   .aggregate(core, artifact, plugin)
+  .settings(name := "sbt-precog-root")
```

Comment:
Worth noting that we *may* have some old infrastructure which hardcodes `root` for the root project. I think I replaced all of that with `/` but I may have missed something.

Reply:
This is just for sbt-precog. I had forgotten about it, tbh. I just did it because IntelliJ is annoying -- every time it reloads the project, it replaces the project name with root's name, so I end up trying to pick which project called "root" I want to load.

Reply:
Wait... what do you mean by replacing with `/`?

Reply:
In sbt commands, `/` is an alias for the root project, whatever it may be named.

Reply:
How can it be an alias for root _and_ a separator for scopes?

Reply:
It's only an alias for root in the shell, and I *think* only in the `project` command. So you can run `project /`, but I don't think you can do something like `//test` and run `test` in root (maybe you can… dunno).